lib/github.sh

```diff
@@ -153,7 +153,6 @@ strap::github::api::token::create() {
     password_attempts=$((password_attempts + 1))
 
     strap::assert::has_length "$status_code" 'Unable to parse GitHub response status.  GitHub response format is likely to have changed.  Please report this to the Strap developers.'
-
```

Comment:
I'd like to keep this newline - it separates assertion logic from the big if-block to follow.